libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1064,7 +1064,7 @@ Json::Value StandardCompiler::compileSolidity(StandardCompiler::InputsAndSetting
 		errors.append(formatErrorWithException(
 			compilerStack,
 			_error,
-			false,
+			Error::Type::InterfaceError,
 			_error.typeName(),
```

Comment:
Since we have an `_error` instance available here, maybe it should be `_error.type()` rather than `Error::Type::InterfaceError`?

Reply:
Makes sense!

Reply:
Actually, looking at it again, a hard-coded `Error::Category::Error` might make more sense there. This is an `Error` that was thrown by the compiler instead of being handled in the usual way so even if it's warning or info, the `Uncaught error` message itself should be an error.

Reply:
What about this one?

Reply:
So you're suggesting to replace `Error::errorSeverity(_error.type())` by `Error::Category::Error`?

Reply:
Yeah. This seems to be reporting the fact that an error was it was thrown instead of being added to the error list in a normal way. And this fact in itself is an error even if the thing that was thrown is warning or info. I think that my original suggestion was wrong.

Reply:
Done